src/main/webapp/index.html

```diff
@@ -16,7 +16,18 @@
   </head>
   <body>
     <div id="content">
-
+      <!--TODO: Add a navbar-->
+      <p id="welcome">Welcome to InterviewMe!</p>
```

Comment:
change id to something more specific? like "welcome-heading"?

Reply:
Done